File: KrasiMilan/QL/src/main/java/nl/uva/softwcons/qls/ast/widget/DefaultStyle.java
package nl.uva.softwcons.qls.ast.widget;

import nl.uva.softwcons.ql.ast.type.Type;
import nl.uva.softwcons.qls.ast.ASTNode;

public class DefaultStyle implements ASTNode {
    private final Type questionType;
    private final Widget widget;

    public DefaultStyle(final Type type, final Widget widget) {
        this.questionType = type;
        this.widget = widget;
    }

    public Type getQuestionType() {
        return questionType;
    }

    public Widget getWidget() {
        return widget;
    }

}

File: CarlaErik/UvA.SoftCon.Questionnaire/QLS/AST/Model/DefaultStyle.cs
using System.Collections.Generic;
using UvA.SoftCon.Questionnaire.Common.AST.Model;
using UvA.SoftCon.Questionnaire.QLS.AST.Model.StyleAttributes;

namespace UvA.SoftCon.Questionnaire.QLS.AST.Model
{
    public class DefaultStyle : QLSNode
    {
        public DataType DataType
        {
            get;
            private set;
        }

        public IEnumerable<StyleAttribute> StyleAttributes
        {
            get;
            private set;
        }

        internal DefaultStyle(DataType dataType, IEnumerable<StyleAttribute> styleAttributes, TextPosition position)
            : base(position)
        {
            DataType = dataType;
            StyleAttributes = styleAttributes;
        }

        public override T Accept<T>(IStyleSheetVisitor<T> visitor)
        {
            return visitor.VisitDefaultStyle(this);
        }
    }
}
